root/edit/components/EditSidebar.js

```diff
@@ -64,9 +63,7 @@ const EditSidebar = ({$c, edit}: Props) => (
         </SidebarProperty>
       )}
 
-      <SidebarDataQuality quality={edit.quality} />
-
-      <SidebarProperty className="" label={l('Requires:')}>
+      <SidebarProperty className="" label={l('For quicker closing:')}>
```

Comment:
New messages should not end with colon:
```suggestion
      <SidebarProperty className="" label={addColonText(l('For quicker closing'))}>
```